.github/renovate.json

```diff
@@ -21,7 +21,6 @@
     },
     {
       "matchDepTypes": ["minor", "patch", "devDependencies"],
-      "automerge": true,
       "automergeType": "pr"
```

Comment:
I just did some more research on this. it seems like this can be left as is and we should add `"ignorePresets": ["helpers:disableRepoForcePush"],` to the top section.